DuggaSys/sectioned.js

```diff
@@ -31,7 +31,8 @@ var isLoggedIn = false;
 // Globals for the automatic refresh (github)
 var isActivelyFocused = false; // If the user is actively focusing on the course page
 var lastUpdatedCodeExampes = null; // Last time code examples was updated
-const UPDATE_INTERVAL = 600 * 100; // Timerintervall for code to be updated (10 minutes)
+var updateMins = 10; // Variable expressed in minutes for how often the code should be updated
```

Comment:
Should be declare as a const instead of var according to best practices:
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let